.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Maybe add those files to your global gitignore.

Reply:
Sure. I am going to remove them in any case.